src/Util.ts

```diff
@@ -85,6 +85,25 @@ export const loadImage = (src: string, imageServerURL?: string | undefined, imag
     });
 };
 
+export const isSvgFile = (src: string) => {
```

Comment:
@hritikaggarwal31 Is this required for your feature?

Reply:
yes.

Reply:
svg is not needed in the dashboard right? We always use png